src/Microsoft.DotNet.XHarness.WebAssembly/xunit-runner.cs

```diff
@@ -0,0 +1,395 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Linq;
+
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+using Xunit.ConsoleClient;
+
+namespace Microsoft.DotNet.XHarness.WebAssembly
+{
+    class MsgBus : IMessageBus
+    {
+        public MsgBus(IMessageSink messageSink)
+        {
+            Sink = messageSink;
+        }
+
+        IMessageSink Sink { get; set; }
+
+        public bool QueueMessage(IMessageSinkMessage message)
+        {
+            return Sink.OnMessage(message);
+        }
+
+        public void Dispose()
+        {
+        }
+    }
+
+    class Discoverer : XunitTestFrameworkDiscoverer
+    {
+        List<ITestClass> _testClasses;
```

Comment:
Would be nice to also make them `readonly` where possible and ideally add `private` too. The .editorconfig supplied with the repo should give you all the hints (at least in VS dogfood does a good bit)

Reply:
addressed